crates/nu-protocol/src/config/mod.rs

```diff
@@ -159,6 +160,8 @@ impl Default for Config {
             use_kitty_protocol: false,
             highlight_resolved_externals: false,
 
+            completion_use_ls_colors: false,
```

Comment:
It would be nice to put this up above with all the other completion items if possible.